pysteps/visualization/utils.py

```diff
@@ -28,6 +37,41 @@
 except ImportError:
     PYPROJ_IMPORTED = False
 
+PYPROJ_PROJ_KWRDS_TO_CARTOPY = {
+    "lon_0": "central_longitude",
+    "lat_0": "central_latitude",
+    "lat_ts": "true_scale_latitude",
+    "x_0": "false_easting",
+    "y_0": "false_northing",
+    "k": "scale_factor",
+    "zone": "zone",
+}
+
+PYPROJ_GLOB_KWRDS_TO_CARTOPY = {
+    "a": "semimajor_axis",
+    "b": "semiminor_axis",
+    "datum": "datum",
+    "ellps": "ellipse",
+    "f": "flattening",
+    "rf": "inverse_flattening",
+}
+
+PYPROJ_PROJECTION_TO_CARTOPY = dict(
```

Comment:
Similarly to above, this dictionary can only be defined if Cartopy is installed.